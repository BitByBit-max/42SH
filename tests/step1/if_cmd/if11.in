if echo hello my name is barbora and this is my group for the 42sh project here at epita;
then echo woo;
fi
