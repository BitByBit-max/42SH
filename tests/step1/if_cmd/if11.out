hello my name is barbora and this is my group for the 42sh project here at epita
woo
